samples/msal-node-samples/AGC-README.md

```diff
@@ -0,0 +1,22 @@
+The non-AGC E2E tests are not able to be run in the AGCE. In order to run - and ONLY run - the AGC E2E tests while in the AGCE, changes must be made to the jest-preset files. Inside of jest-preset.js and jest-preset-no-setup.js, line 7 must be commented and line 9 must be uncommented.
```

Comment:
Referencing line numbers in files like this aren't super durable, suggest explicitly calling out the changes here vs the line numbers and ensuring that comments in jest-preset* files adequately explain what is needed to get this to work.